grammar.js

```diff
@@ -228,7 +232,14 @@ module.exports = grammar({
 
         payee: $ => /[^(*!\n][^*!\n]*/,
 
-        note: $ => seq(';', /.*/),
+        note: $ => seq(
```

Comment:
This might be incompatible with a proper implementation for tags (see https://github.com/cbarrete/tree-sitter-ledger/issues/13). I believe that an external scanner would need to handle *all* the kinds of notes.

Reply:
I'm happy to merge it for now though as it's better than nothing. Just rebase it and I'll merge it.

Reply:
I agree that something is better than nothing. My experiments with add support for tags also lead to the question whether an external scanner might be better, but I wanted to look at other tree-sitter grammar first for inspiration of other ways to possibly approach this.